docs/admin/kubeadm.md

```diff
@@ -28,7 +28,7 @@ following steps:
 1. Outputting a kubeconfig file for the kubelet to use to connect to the API
```

Comment:
since we have "the kubelet" here, we should maintain consistency and have it throughout.